src/libs/data-utils.js

```diff
@@ -293,20 +294,22 @@ export const EntityUploader = class EntityUploader extends Component {
             onClick: () => this.doUpload()
           }, ['Upload'])
         }, [
-          'The first column header must be:',
-          div({ style: { fontFamily: 'monospace', margin: '0.5rem', fontWeight: '600' } }, ['entity:[type]_id']),
-          'where ',
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['[type]']),
-          ` is the desired name of the data table in ${getAppName()}. For example, to create or update a `,
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['participant']),
-          ' table use ',
-          span({ style: { fontFamily: 'monospace', fontWeight: '600' } }, ['entity:participant_id']),
-          div({ style: { margin: '0.5rem 0 1rem' } }, ['All of the values in the ID column must be unique.']),
+          div({
+            style: { padding: '0 0 1rem' }
+          }, [
+            'Choose the data import option below. For more info on the table',
+            h(Link, {
+              'aria-label': 'Click for table info',
+              href: 'https://support.terra.bio/hc/en-us/articles/360025758392'
+            }, [
+              ' click here.'
+            ])
```

Comment:
I'd condense this into one line, maybe make the whole sentence a link? Like, 'Click here for more info on the table.`